docs/resize-rbd.md

```diff
@@ -0,0 +1,277 @@
+# Resize RBD PVCs
+
+- [Resize RBD PVCs](#resize-rbd-pvcs)
```

Comment:
Add Minimum kubernetes version required to resize filesystem and block  PVC

Reply:
> kubernetes version running in your cluster should be >= v1.16

it is there (now)